core/frontend/src/tile/RealityModelTileTree.ts

```diff
@@ -751,13 +752,13 @@ class RealityTreeReference extends RealityModelTileTree.Reference {
     // If a type is specified, display it
     if (type !== undefined) {
       switch (type.toUpperCase()) {

```

Comment:
type should bot be change to upper case if string in cases are not upperc ase.

Reply:
Removed the `.toUpperCase` issue.